### PR TITLE
Move lines to `geometry`

### PR DESCRIPTION
**Pull request by schmidma**
What it says, just moving

### CODE REVIEW

File: crates/geometry/src/line.rs [hunk 1]
```diff
@@ -143,9 +144,10 @@ impl<Frame, const DIMENSION: usize> Line<Frame, DIMENSION> {
     }
 }
 
-impl<From, To, const DIMENSION: usize> Mul<Line<From, DIMENSION>> for Isometry<From, To, DIMENSION>
+impl<From, To, const DIMENSION: usize, Inner> Mul<Line<From, DIMENSION>>
+    for Transform<From, To, Inner>
 where
-    Isometry<From, To, DIMENSION>: Mul<Point<From, DIMENSION>, Output = Point<To, DIMENSION>>,
+    Self: Mul<Point<From, DIMENSION>, Output = Point<To, DIMENSION>> + Copy,
```
Comment (knoellle) [inline crates/geometry/src/line.rs:150]: Isn't this the same change as in #780?

File: crates/geometry/src/line.rs [hunk 2]
```diff
@@ -17,6 +17,7 @@ pub struct Line<Frame, const DIMENSION: usize>(
 );
 
 pub type Line2<Frame> = Line<Frame, 2>;
+pub type Line3<Frame> = Line<Frame, 3>;
```
Comment (knoellle) [inline crates/geometry/src/line.rs:20]: Is this used anywhere?

**Review by knoellle — CHANGES_REQUESTED** [review]
I like it but have some minor remarks

File: crates/geometry/src/line.rs (hunk 1 above)
Comment (schmidma) [reply crates/geometry/src/line.rs]: True, I reverted this change here. Should be the same as in main now...

File: crates/geometry/src/line.rs (hunk 2 above)
Comment (schmidma) [reply crates/geometry/src/line.rs:20]: No, just there for completeness ;) Did not hurt, and the next person is lucky

**Review by knoellle — APPROVED** [review]
LGTM

**Review by knoellle — APPROVED** [review]
LGTM

**Pull request merged**